composer.json

```
@@ -114,6 +114,7 @@
     "drupal/roleassign": "^2",
     "drupal/simple_oauth": "^5",
     "drupal/tfa": "^2",
+    "drupal/smtp": "^1",
```

@baldwinlouie The item should be the alphabetical order:

```suggestion
    "drupal/smtp": "^1",
    "drupal/tfa": "^2",
```

---

@naoi This has been updated.